.tool-versions

```diff
@@ -0,0 +1,2 @@
+erlang latest
```

Comment:
"What is this file?"
A fairly common pattern added in https://github.com/tolemi-inc/tournament/pull/2/commits/b91d3729287825c1b8a14048b76676d5afa20976

Along with this I'd update the developer getting started steps, likely found in a `CONTRIBUTING.md` or `README.md` or wiki.